packages/expo-facebook/src/Facebook.types.ts

```diff
@@ -0,0 +1,109 @@
+export type FacebookAuth = {
```

Comment:
I find this type name ambiguous: are you referring to authentication or authorization?